src/plugins/notebook/components/popup-menu.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+<div class="l-browse-bar__view-switcher c-ctrl-wrapper c-ctrl-wrapper--menus-left">
```

Comment:
Appears to be using a view-switcher class?

Reply:
ahh, copy paste error :P

Reply:
removed extra HTML elements.